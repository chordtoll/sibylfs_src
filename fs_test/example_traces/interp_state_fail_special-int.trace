@type script
mkdir /foo 0o10000
