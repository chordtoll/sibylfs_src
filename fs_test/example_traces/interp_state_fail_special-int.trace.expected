# processing file 'example_traces/interp_state_fail_special-int.trace' ...
        @type trace
     2: mkdir "/foo" 0o10000
# 
# Fatal error:      2: mkdir "/foo" 0o10000
#    special states found:
#      - OS_special (Implementation_defined, "mkdir: additional file permission bits")
#    no normal result states
# processing file 'example_traces/interp_state_fail_special-int.trace' failed
